src/react/components/common/assetPreview/videoAsset.tsx

```diff
@@ -117,7 +117,12 @@ export class VideoAsset extends React.Component<IVideoAssetProps> {
     }
 
     public componentDidMount() {
-        this.videoPlayer.current.subscribeToStateChange(this.onVideoStateChange);
+        if (this.props.autoPlay) {
```

Comment:
Can you add a unit test to validate this behavior?

Reply:
Done